app/models/review_request.rb

```diff
@@ -24,7 +24,7 @@
 #  fk_rails_...  (reviewer_id => users.id)
 #
 class ReviewRequest < ApplicationRecord
-  enum state: { active: 'active', removed: 'removed', reviewed: 'reviewed' }
+  enum state: { active: 'active', removed: 'removed' }
```

Comment:
Why did we remove that state?

Reply:
I believe it is unecessary, we can check that by asking if that review request has at least one review.